### PR TITLE
cleaning css files

### PR DESCRIPTION
**Pull request by ghost**
trying to reduce css code for a better readability and less code to ship and download

replaces #14510 since it is broken.
may replace #14500 

please review when WIP status is resolved

### CODE REVIEW

**Comment by BO41** [comment]
please review, merge and maybe force some code standards for css files :)

File: setup/styles.css [hunk 1]
```diff
@@ -89,19 +89,19 @@ h1 {
...
+    margin-right: .5em;
     text-decoration: none;
     font-size: small;
 }
 
 /* phpMyAdmin logo colors */
 
 .blue {
-    color: #666699;
+    color: #669;
```
Comment (ibennetch) [inline setup/styles.css]: I, personally, prefer the six digit format to the three digit one. I'm not sure about anyone else, though, maybe I'm alone in that opinion.

Comment (williamdes) [reply setup/styles.css]: @ibennetch Nop, I also think we should use 6 digit notation.

Comment (BO41) [reply setup/styles.css]: okay ;) will change it

**Comment by BO41** [comment]
@ibennetch @williamdes changed it in all files and also where it was in the 3 digit notation before.
(also found some mistakes I made :fearful: )

**Comment by BO41** [comment]
Codacy/PR Quality Review fails but I did nothing wrong. The changes should be fine

**Comment by williamdes** [comment]
@BO41 Codacy is set up but no one seems to be interested by the reported errors/warnings, maybe I am wrong ?
cc @ibennetch @nijel

**Comment by BO41** [comment]
closing for now because of inactivity.
reopen if you want to merge

**Pull request closed without merging**